docs/sphinx/source/whatsnew/v0.9.3.rst

```diff
@@ -5,7 +5,7 @@ v0.9.3 (TBD)
 
 Deprecations
 ~~~~~~~~~~~~
-
+  * Removed the kwarg `close` from `pd.date_range` in the examples since it is deprecated for pandas >= 1.4.0. (:pull:`1540`)
```

Comment:
```suggestion
  * Removed the kwarg ``closed`` from ``pd.date_range`` in the examples since it is deprecated for pandas >= 1.4.0. (:pull:`1540`)
```